src/Components/Scales/scale.tsx

```diff
@@ -79,13 +84,23 @@ export const Scale = (scalesProps: IUsersProps) => {
               onClick={(event: any) => {
                 setScaleId(scale?.id as string)
                 setDisplayScale(true)
+                handleClick(event)
               }}
             >
-              <Icon style={{ width: '25px' }} src={String(EyeScaleIcon)} />
+              <Icon style={{ width: '25px' }} src={String(BulletPointIcon)} />
             </IconButton>
           </CardActions>
         </CardContent>
       </Card>
+      { open ?
```

Comment:
Aqui não está errado, porém, eu criei uma função que vai omitir essa lógica de abrir ou não (busquei uma forma de centralizar a lógica). Deixa só um To Do pra mim colocar depois.